crypto/ec/ecp_nistp224.c

```diff
@@ -1410,9 +1413,6 @@ int ec_GFp_nistp224_points_mul(const EC_GROUP *group, EC_POINT *r,
     const EC_POINT *p = NULL;
     const BIGNUM *p_scalar = NULL;
 
-    if (ctx == NULL)
-        if ((ctx = new_ctx = BN_CTX_new()) == NULL)
-            return 0;
```

Comment:
In such case you can remove new_ctx altogether. This comment applies even below.